Dockerfile.autorest

```diff
@@ -0,0 +1,23 @@
+FROM registry.access.redhat.com/ubi8/nodejs-14
```

Comment:
This will not be LTS after 2021-10-26 which will be nodejs-16. Nodejs 14 will be in maintenance but maybe a comment to change this image or something?